tensorrt_llm/_torch/modules/linear.py

```diff
@@ -843,6 +848,7 @@ def load_weights_vanilla(self, module: Linear, weights: List[Dict]) -> None:
         E2M1_MAX = 6.0
         module.inv_input_scale.data = module.input_scale / E2M1_MAX
         copy_weight(module.alpha, alpha)
+        module.scalar_alpha = alpha.item()
```

Comment:
I do not understand this. If there is already a module.alpha has the info, do we really need a `scalar_alpha`, can it be derived?

Reply:
module.alpha is a gpu tensor with 1 element. In the cute_dsl kernel I make this parameter as float directly (use floa tis more simple than tensor). So I add the scalr_alpha here.